vault/templates/index.html

```diff
@@ -3,12 +3,20 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
-    <title>repl.it</title>
+    <title>Karma Computing Vault</title>
     <link href="style.css" rel="stylesheet" type="text/css" />
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
 
   </head>
   <body>
+  {% with messages = get_flashed_messages() %}  
+         {% if messages %}  
+               {% for message in messages %}  
+                    <p class="alert alert-danger" role="alert">
+                    <a href="https://plans.karmacomputing.co.uk/plan/50a99a74-31b2-4e6c-a5d2-15b1bfe0832d/WordPress%20Hack%20Prevention"> {{ message }}</a></P>
```

Comment:
Please set in config as `SIGN_UP_URL`  then get with `app.config["SIGN_UP_URL"]` because this is configuration not code